tests/installation/setup_libyui.pm

```diff
@@ -13,9 +13,10 @@ use Utils::Backends;
 use Utils::Architectures;
 use testapi;
 use YuiRestClient;
+use version_utils 'is_sle';
 
 sub run {
-    die 'Module requires YUI_REST_API variable to be set.', unless get_var('YUI_REST_API');
+    die 'Leap 15.2 and below and SLE 15-SP2 and below do not have libyui-rest, exit now.' if (is_sle('<=15-SP2') || is_leap('<=15.2') || get_var('YUI_REST_API'));
```

Comment:
we should not die if `get_var('YUI_REST_API')` returns true.
Maybe negate the condition?

Reply:
done

Reply:
Now that I think about it, in case the YUI_REST_API variable is not there, the message `Leap 15.2 and below and SLE 15-SP2 and below do not have libyui-rest, exit now.` is not very helpful to the user.
I think that it would be better to have it as you originally had it, first checking for the `YUI_REST_API` variable, and then checking the version. At least this is how I think about it.

Reply:
How about this: 'The SUT version is below Leap 15.2 or SLE 15-SP2, or the variable YUI_REST_API is not set, exit now.'

Reply:
For me this is more complicated, it should give an error that corresponds to each case. Still let's see what others say

Reply:
2 `die`s is preferable for sure.

Reply:
re-committed